s3cmd

```diff
@@ -2135,6 +2135,47 @@ def cmd_dellifecycle(args):
     output(u"%s: Lifecycle Policy deleted" % uri)
     return EX_OK
 
+def cmd_setnotification(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    uri = S3Uri(args[1])
+    notification_policy_file = args[0]
+
+    with open(deunicodise(notification_policy_file), 'r') as fp:
+        notification_policy = fp.read()
+
+    if cfg.dry_run:
+        return EX_OK
+
+    response = s3.set_notification_policy(uri, notification_policy)
+
+    debug(u"response - %s" % response['status'])
+    if response['status'] == 200:
+        output(u"%s: Notification Policy updated" % uri)
+    return EX_OK
+
+def cmd_getnotification(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    uri = S3Uri(args[0])
+
+    response = s3.get_notification_policy(uri)
+
+    output(u"%s" % getPrettyFromXml(response['data']))
```

Comment:
I might be wrong, but I don't think that you need the `u"%s" %` if you just want to output the `getPrettyFromXml(...)`.